packages/console/src/components/ImagePlaceholder/index.module.scss

```diff
@@ -1,6 +1,9 @@
+@use '@/scss/underscore' as _;
+
 .container {
   background: rgba(#ffcc4d, 0.3);
   display: flex;
   justify-content: center;
   align-items: center;
+  padding: _.unit(2);
```

Comment:
I don't think a placeholder component should have padding by itself

Reply:
https://silverhand-io.slack.com/archives/C0243P3E4KS/p1649321315314789